content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -523,6 +523,12 @@ The presence of this annotation on a Job indicates that the control plane is
 [tracking the Job status using finalizers](/docs/concepts/workloads/controllers/job/#job-tracking-with-finalizers).
 You should **not** manually add or remove this annotation.
 
+{{< note >}}
+Starting from Kubernetes 1.26, this annotation is marked as deprecated.
+Kubernetes 1.27 or newer will ignore this annotation and always track Jobs
+using finalizers.
```

Comment:
Maybe we want to delete these because we cannot predict the future.

Reply:
I want to give clear expectations on when the semantics for the annotation will not be honored, so people get fair notice.

Reply:
We do [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future), with an exception for confirmed deprecation and removals.

Is this a commitment that SIG Architecture would confirm (if asked)?

Reply:
I suppose you meant SIG Apps, and yes, this was approved in the KEP
https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2307-job-tracking-without-lingering-pods#deprecation

Reply:
cc @liggitt @johnbelamaric @derekwaynecarr

Reply:
The annotation was for bookkeeping between the API server and controller during the transition while the feature was in development and graduating. I think it's kind of unfortunate it ended up being documented here at all.

It sounds good to me to say in a given Kubernetes version range the annotation indicates which jobs are tracked using finalizers while the feature was under development, and in Kubernetes 1.27+ will be ignored and will not be set by the API server because all jobs will be tracked using finalizers.

Reply:
sounds good.
I switched the wording from `or` to `and` to make it sound like a fact instead of a hypothesis.

Reply:
@liggitt how does it look now?

Reply:
lgtm